Src/Add-AzureDtlVM.ps1

```diff
@@ -1,64 +1,142 @@
+<#
+.SYNOPSIS 
+    This script adds the specified number of Azure virtual machines in the specified DevTest Lab.
+
+.DESCRIPTION
+    It allows the creation of the VMs inside a specific lab. This script needs to be scheduled for the creation of the VMs of each lab. 
+
+.PARAMETER LabName
+    Mandatory. Name of Lab.
+
+.PARAMETER VMCount
+    Mandatory. Number of VMs to create with this execution.
+
+.PARAMETER ImageName
+    Mandatory. Name of base image in lab.
+
+.PARAMETER TotalLabSize
+    Mandatory. Desired total number of VMs in the lab.
+
+.PARAMETER BatchSize
+    Optional. How many VMs to create in each batch.
+    Default 30.
+
+.PARAMETER TemplatePath
+    Optional. Path to the Deployment Template File.
+    Default ".\dtl_multivm_customimage.json".
+
+.PARAMETER ShutdownPath
+    Optional. Path to the Shutdown file.
+    Default ".\dtl_shutdown.json".
+
+.PARAMETER Size
+    Optional. Size of VM image.
+    Default "Standard_A2_v2".
+
+.PARAMETER VMNameBase
+    Optional. Prefix for new VMs.
+    Default "vm".
+
+.PARAMETER VNetName
+    Optional. Virtual Network Name.
+    Default "dtl" + $LabName.
+
+.PARAMETER SubnetName
+    Optional. Subnet Name.
+    Default "dtl" + $LabName + "SubNet".
+
+.PARAMETER location
+    Optional. Location for the Machines.
+    Default "westeurope".
+
+.PARAMETER TimeZoneId
+    Optional. TimeZone for machines.
+    Default "Central European Standard Time".
+
+.PARAMETER profilePath
+    Optional. Path to file with Azure Profile.
+    Default "$env:APPDATA\AzProfile.txt".
+
+.PARAMETER DaysToExpiry
+    Optional. How many days before expiring the VMs (-1 never, 0 today, 1 tomorrow, 2 ...) Defaults to tomorrow.
+    Default "1".
+
+.PARAMETER ExpirationTime
+    Optional. What time to expire the VMs at. Defaults to 3am. In form of 'HH:mm' in TimeZoneID timezone.
+    Default "03:00".
+
+.PARAMETER ShutDownTime
+    Optional. Shutdown time for the VMs in the lab. In form of 'HH:mm' in TimeZoneID timezone.
+    Default $ExpirationTime.
+
+.EXAMPLE
+    Add-AzureDtlVM -LabName University -VMCount 50 -ImageName "UnivImage" -TotalLabSize 200
+
+.EXAMPLE
+    Add-AzureDtlVM -LabName University -VMCount 15 -ImageName "UnivImage" -TotalLabSize 15 -ExpirationTime "16:00" -DaysToExpiry 0
+
+.EXAMPLE
```

Comment:
Add an example of running from Azure Automation. Look at the automation variables and pass those as parameters.